Makefile

```diff
@@ -11,7 +11,10 @@ format:
 	(./code/gradlew -p code/$(EXTENSION-LIBRARY-FOLDER-NAME) spotlessApply)
 	(./code/gradlew -p code/$(TEST-APP-FOLDER-NAME) spotlessApply)
 
-format-check:
+format-license:
+	(./code/gradlew -p code licenseFormat)
```

Comment:
It doesn't appear you ran this as I don't see the header applied to your scripts. As you are including this here in this PR, can you run it to update the copyright headers?

Reply:
I will apply the header in the next PR.  It will update many files, I would like keep them separated than this PR.